app/src/test/java/com/keylesspalace/tusky/ComposeActivityTest.kt

```diff
@@ -119,15 +98,27 @@ class ComposeActivityTest {
         })
 
         val instanceDaoMock = mock(InstanceDao::class.java)
+        `when`(instanceDaoMock.loadMetadataForInstance(any())).thenReturn(
+                Single.just(InstanceEntity(instanceDomain, emptyList(),null, null, null, null))
+        )
+
         val dbMock = mock(AppDatabase::class.java)
         `when`(dbMock.instanceDao()).thenReturn(instanceDaoMock)
 
-        activity.mastodonApi = apiMock
-        activity.accountManager = accountManagerMock
-        activity.database = dbMock
+        val viewModel = ComposeViewModel(
```

Comment:
Idk if it''s a good idea to have a real ViewModel but activity test now is more "integration" one in any case. We probably need some `ComposeViewModel` tests as well but there's barely any logic there so it's not critical.